.github/workflows/release.yml

```diff
@@ -119,7 +119,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: "lts/*"
+          node-version: "16.x"
```

Comment:
v16 is where we have the minimum npm version that can run `npm audit signatures`, so this needs to be above 14